accel-pppd/ctrl/sstp/sstp.c

```diff
@@ -2607,6 +2587,8 @@ static void ssl_load_config(struct sstp_serv_t *serv, const char *servername)
 #else
 			DH *dh;
 
+			in = BIO_new(BIO_s_file());
```

Comment:
There was a bug, when `ssl-pemfile` is not set, but `ssl-dhparam` is. in this case,  `in` here will be NULL. Fixed.